src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift

```diff
@@ -27,6 +27,14 @@ class HomeStatisticsCellModel {
 					.compactMap { cardID in
 						statistics.linkCards.first { $0.header.cardID == cardID }
 					}
+				#if DEBUG
+				if isUITesting {
```

Comment:
Can we please wrap this into a function something like setupMockLinkCards and call it on line 54?